helm/templates/server/statefulset.yml

```diff
@@ -109,6 +109,7 @@ spec:
         {{- end }}
       restartPolicy: Always
       serviceAccountName: {{ include "pinot.server.serviceAccountName" . }}
+      terminationGracePeriodSeconds: {{ .Values.server.terminationGracePeriodSeconds }}
```

Comment:
Does this mean the terminationGracePeriod passed from values.yaml wasn't taking effect earlier?